app/templates/frameworks/services.html

```diff
@@ -65,91 +64,115 @@ <h2 class="summary-item-heading">{{ framework.name }} is closed for applications
+      <h2 class="govuk-heading-m">Complete services</h2>
   {% elif framework.status == 'pending' or 'standstill' %}
     {% if declaration_status == 'complete' %}
-      {{ summary.heading("Submitted services") }}
+        <h2 class="govuk-heading-m">Submitted services</h2>
     {% else %}
-      {{ summary.heading("Completed services") }}
+        <h2 class="govuk-heading-m">Completed services</h2>
     {% endif %}
   {% endif %}
-  {% call(draft) summary.list_table(
-    complete_drafts,
-    caption="Complete services",
-    empty_message="You haven’t marked any services as complete yet." if framework.status == 'open' else "You didn’t mark any services as complete.",
-    field_headings=[
-        "Service name",
-        "Progress",
-        "Make a copy"
-    ],
-    field_headings_visible=False
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(
-          draft.serviceName,
-          url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id)
-      ) }}
-      {% if framework.status == 'open' %}
-        {% call summary.field(action=True) %}
-          <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
-            <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
-            {{ govukButton({
-              "text": "Make a copy",
-              "classes": "govuk-!-margin-0",
-            }) }}
-          </form>
-        {% endcall %}
-      {% else %}
-        {{ summary.text() }}
-      {% endif %}
-    {% endcall %}
-  {% endcall %}
 
-  <hr class="govuk-section-break govuk-section-break--m">
+  {% if not complete_drafts %}
+  {{ "You haven’t marked any services as complete yet." if framework.status == 'open' else "You didn’t mark any services as complete."  }}
+  {% endif %}
+  {% set ns = namespace(completed_rows = []) %}
+   {% if framework.status == 'open' %}
+    {% for complete_draft in complete_drafts %}
+    {% set service_link %}
+    <a class="govuk-link" href="{{ url_for('.view_service_submission', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">{{ complete_draft.serviceName }}</a>
+    {% endset %}
+    {% set make_copy_button %}
+    <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=complete_draft.lot, service_id=complete_draft.id) }}">
+        <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
+        {{ govukButton({
+          "text": "Make a copy",
+          "classes": "govuk-!-margin-0",
+        }) }}
+    </form>
+    {% endset %}
+        {% set row = ns.completed_rows.append(
+        [
+          {"html": service_link},
+          {"html": make_copy_button}
+        ]
+    ) %}
+        {{ govukTable({
+      'head': [
+        {'text': "Service name"},
+        {'text': "Make a copy"}
```

Comment:
Because there are only two items in this table it does not visually align with the draft table as before.
## Before
![Screenshot 2021-06-17 at 15 32 39](https://user-images.githubusercontent.com/1764158/122417254-45942680-cf81-11eb-9946-414847938c49.png)

## After
![Screenshot 2021-06-17 at 15 31 41](https://user-images.githubusercontent.com/1764158/122417133-2d240c00-cf81-11eb-9d60-c58995d6b8e9.png)

Reply:
How do we feel about right-aligning the buttons?

We've done that previously with a class: https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/afedd090199322b0dfe1545b824e10e03ae7899e/app/templates/buyers/dashboard.html#L61

Reply:
@domoscargin thanks for the suggestion, done now in 5646f92